lib/matplotlib/tri/tricontour.py

```diff
@@ -239,7 +251,22 @@ def _contour_args(self, args, kwargs):
 
 xunits, yunits : registered units, optional
     Override axis units by specifying an instance of a
-    :class:`matplotlib.units.ConversionInterface`.""")
+    :class:`matplotlib.units.ConversionInterface`.
+
+antialiased : bool, optional
+    Enable antialiasing, overriding the defaults.  For
+    filled contours, the default is *True*.  For line contours,
+    it is taken from :rc:`lines.antialiased`.
+
+nchunk : int >= 0, optional
+    If 0, no subdivision of the domain.  Specify a positive integer to
+    divide the domain into subdomains of *nchunk* by *nchunk* quads.
+    Chunking reduces the maximum length of polygons generated by the
+    contouring algorithm which reduces the rendering workload passed
+    on to the backend and also requires slightly less RAM.  It can
+    however introduce rendering artifacts at chunk boundaries depending
+    on the backend, the *antialiased* flag and value of *alpha*.""")
+
```

Comment:
```suggestion
```
There should only be two empty lines between functions.

This causes flake8 to complain: "E303 too many blank lines (3)".